### DISCUSSION TITLE
SimpleGrid not responsive

### DISCUSSION THREAD

**Discussion by aldwnesx** [chakra-ui/chakra-ui#4662] [Q&A]
I have a SimpleGrid with two Box elements (they wrap another component) that and have set up as 100% width and 100% height.
I only display 2 columns in a row. I would like to break it into two rows if screen gets smaller. However, it's only minimizing my child elements instead of rendering them on separate rows

https://chakra-ui.com/docs/layout/simple-grid#changing-the-spacing-for-columns-and-rows

```

<SimpleGrid minChildWidth="120px" spacing="40px">
        <Box width="100%"  >  <componentTwo/> </Box>
        <Box width="100%"  >  <componentThree/></Box>

</SimpleGrid>
```

**Comment by payapula** [comment]
Hi @aldwnesx 

I tried the example you provided and it looks like the simplegrid is working as you mentioned.

Take a look at the example here - https://codesandbox.io/s/solitary-shape-xefje?file=/src/index.js and let me know if this is what you expect. I just added background colors to check if grid is working.

**Reply by aldwnesx** [reply]
> Hi @aldwnesx
> 
> I tried the example you provided and it looks like the simplegrid is working as you mentioned.
> 
> Take a look at the example here - https://codesandbox.io/s/solitary-shape-xefje?file=/src/index.js and let me know if this is what you expect. I just added background colors to check if grid is working.

Yes, this is what I'm looking for but it's not working for me. Could it be the components that I am using? I have not much control over them as they're obtained from other packages

**Reply by payapula** [reply]
yes, it could be, due to their styles. Try to override their styles like position/float properties. 

If possible, please try to create a reproduction with those components and post it here so that we can look and work it on.

Thanks.
